src/main/java/supersymmetry/api/fluids/SusyFluidTypes.java

```diff
@@ -4,8 +4,6 @@
 import gregtech.api.fluids.fluidType.FluidTypeLiquid;
 
 public class SusyFluidTypes {
-
-    public static final FluidType IMPURE_SLURRY = new FluidTypeLiquid("impure_slurry", "impure_slurry", null, "fluid.slurry.impure");
-    public static final FluidType PURE_SLURRY = new FluidTypeLiquid("pure_slurry", "pure_slurry", null, "fluid.slurry.pure");
-
+    public static final FluidType IMPURE_SLURRY = new FluidTypeLiquid("impure_slurry", "impure_", "_slurry", "fluid.slurry.impure");
+    public static final FluidType PURE_SLURRY = new FluidTypeLiquid("pure_slurry", null, "_slurry", "fluid.slurry.pure");
```

Comment:
Make sure to include the prefix of "pure_" in this line!

Reply:
in the base modpack, pure slurries are acknowledged as just the material followed by _slurry, there is no pure